### PR TITLE
Config innlasting/ codePipeline

### PR DESCRIPTION
**Pull request by gregorgabriel**
refactoring angående innlasting av Config 
endring til CodePipleine for dependabot

### CODE REVIEW

**Review by anetteOlli — APPROVED** [review]

File: pipeline.yml [hunk 1]
```diff
@@ -59,13 +59,18 @@ Parameters:
...
-  CodeStarConnectionArn:
-    Type: AWS::SSM::Parameter::Value<String>
-    Default: '/github-connection'
-    Description: The ARN for the CodestarConnection that connects to the Git (BitBucket) repo
+  GitHubOAuthToken:
+    Description: OAuth token used by AWS CodePipeline to connect to GitHub
+    NoEcho: true
+    Type: String
+    Default: '{{resolve:secretsmanager:githubtoken:SecretString}}'
+    GitHubOwner:
```
Comment (nraanes) [inline pipeline.yml:67]: For mye innrykk

File: pipeline.yml [hunk 2]
```diff
@@ -368,14 +371,15 @@ Resources:
             - Name: Source
               ActionTypeId:
                 Category: Source
-                Owner: AWS
+                Owner: ThirdParty
                 Version: '1'
                 Provider: CodeStarSourceConnection
```
Comment (nraanes) [inline pipeline.yml:376]: Provider: GitHub

**Review by nraanes — CHANGES_REQUESTED** [review]
Se kommentarer

File: pipeline.yml (hunk 1 above)
Comment (gregorgabriel) [reply pipeline.yml:67]: Sorry, I dont see it:
`  GitHubOAuthToken:
    Description: OAuth token used by AWS CodePipeline to connect to GitHub
    NoEcho: true
    Type: String
    Default: '{{resolve:secretsmanager:githubtoken:SecretString}}'
    GitHubOwner:
      Description: GitHub username owning the repo
      Type: String
      Default: BIBSYSDEV
  GitRepo:`

Comment (gregorgabriel) [reply pipeline.yml:67]: you are right

**Pull request merged**